docs/release-notes/8.0.asciidoc

```diff
@@ -1,6 +1,86 @@
 [[release-notes-header-8.0.0]]
 == 8.0
 
+[discrete]
+[[release-notes-8.0.0]]
+=== 8.0.0
+
+[discrete]
+[[known-issues-8.0.0]]
+==== Known issues
+*Case migration errors might be logged when upgrading*
+
+You might find the `Failed to migrate user action alerts` error message in your {kib} migration logs when upgrading to {stack} version 8.0.0. This error is incorrectly logged when migrating cases and can be ignored ({pull}124950[#124950]).
+
+Here is an example of an error message you might encounter:
+
+[code block]
+----
+[2022-02-07T20:25:58.614+00:00][ERROR][savedobjects-service] Failed to migrate user action alerts with doc id: 7420fe08-c2ed-51d2-b077-46deb4bf76c9 version: 8.0.0 error: Unexpected token in JSON at position 0
+----
+
+*Existing or new rules that use the legacy alerts index may temporarily fail after upgrading*
+
+After you upgrade to {stack} version 8.0.0, existing and new rules might fail to execute  if their source index is configured to use a legacy alert index pattern created in {stack} version 7.x (`.siem-signals-<space-id>`). Rule failures will likely cause coverage gaps, which will be proportional in time to the scheduled interval of the rule. Rules will start to successfully execute after legacy alerts are no longer within the scheduled time period queried by the rule. Despite this automatic correction, coverage gaps (up to about a minute interval) might still remain ({pull}124327[#124327]).
+
+[discrete]
+[[upgrade-reqs-8.0.0]]
+==== Upgrade requirements
+* Rules must be manually disabled before upgrading to {stack} version 8.0.0, and then manually enabled after the process completes. Refer to [Upgrade to Elastic Security] for more information.
+* To make the *Data view* option appear in an environment with legacy alerts, a user with elevated role privileges must visit the {es-sec-app}, open a page that displays alert data (such as the Overview page), then refresh the page. The user's role privileges must allow them to enable the detections feature in a Kibana space. Refer to <<enable-detections-ui, Enable and access detections>> for more information.
+
++
+NOTE: If new alerts are generated in an upgraded environment without legacy alerts, refreshing any page with alert data in {elastic-sec} will make the *Data view* option appear in the {es-sec-ui}.
+
+* The system index for detection alerts has been renamed from `.siem-signals-*` to `.alerts-security.alerts-*`. As such, the schema used for alert documents in {elastic-sec} has changed. Users that access documents in the `.siem-signals` indices will be able to modify their API queries and scripts to operate properly on the new 8.0 alert documents.
```

Comment:
Suggestions in **_bold italics:_**
* The system index for detection alerts has been renamed from `.siem-signals-*` to `.alerts-security.alerts-*` **_and is now a hidden index_**. As such, the schema used for alert documents in {elastic-sec} has changed. Users that access documents in the `.siem-signals` indices **_via API_** ~~will be able to~~ **_must_** modify their API queries and scripts to operate properly on the new 8.0 alert documents.   **_The new schema is here {link to new schema doc section}._**